lib/Agrammon/Preprocessor.pm6

```diff
@@ -59,6 +59,12 @@ sub preprocess(Str $source, %options --> Str) is export {
         if $content.starts-with('?') {
             @result-lines.push('');
             given $content {
+                when /^ '?ifnot' \h+ <option=.ident> \h* $/ {
+                    my $enabled = not so %options{~$<option>};
+                    my $matched = $enabled;
+                    my $start-line = $number + 1;
+                    @open.push(OpenDirective.new(:$start-line, :$enabled, :$matched, :accept-elsif));
+                }
                 when /^ '?if' \h+ <option=.ident> \h* $/ {
                     my $enabled = so %options{~$<option>};
```

Comment:
I think I'd have preferred this as `?if !A`, because then we can also support `?elsif !A` too without having to introduce an `?elsifnot` which is a bit of a mouthful, plus it's easier to remember two features that can be combined than their product. Also, I think it'd be less code; the existing `when` becomes:

```
when /^ '?if' \h+ [$<negate>='!']? <option=.ident> \h* $/ {
    my $enabled = $<negate> ?? .not !! .so given %options{~$<option>};
```

Reply:
I see, much better! I'll change it tomorrow...